code/style.css

```diff
@@ -1,5 +1,78 @@
-/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
-
 body {
-  background-color: lightblue;
+    margin: 50px;
+    text-align: center;
+    font-family: 'Rubik', sans-serif;
+}
+
+nav a {
+    font-family: 'Rubik', sans-serif; 
+    text-transform: uppercase;
+    letter-spacing: 3px;
+    padding: 10px;
+}
+
+a:link, a:visited {
+    color: rgb(38, 38, 41);
+    text-decoration: none;
+}
+
+a:hover, a:active {
+    color: blue;
+    text-decoration: line-through;
```

Comment:
I like the hover effect but it confuses me a little bit. Maybe underline would make more sense?